dvc/ui/__init__.py

```diff
@@ -260,6 +263,23 @@ def isatty(self) -> bool:
 
         return sys.stdout.isatty()
 
+    def open_browser(self, rel_parent_path, file: str = "index.html"):
```

Comment:
Not sure we should provide `rel_parent_path` and `file` separately:
The assumption that our webpages are created as `{path}/index.html` is a concept from our visualization code, UI should not be aware of this. I think we would be fine just providing a `path` (or rather `relpath` if we want to nudge user in proper direction.)

Reply:
I'll add a few requirements here to the `path`/`file` parameter here: 
1. It should accept both `os.Pathlike` and `str`.
2. The path could be either relative, absolute, or a non-normalized path (eg: `src/../index.html`) that we pass straight through CLI arguments.

Reply:
I guess that `open_browser` could just accept any kind of path and change the logic so that, for the WSL case, the given path is converted to relative.

Reply:
>     1. It should accept both `os.Pathlike` and `str`.

👍 

>     2. The path could be either relative, absolute, or a non-normalized path (eg: `src/../index.html`) that we pass straight through CLI arguments. The reason for the non-normalized path is that
>        `Path("/src/../index.html").as_uri()` will just create file uri like `file:///src/../index.html`.

Do you mean that we should normalize the path inside `open_browser`?

Reply:
> Do you mean that we should normalize the path inside open_browser?

yes, `Path(path).resolve()` should probably work.